Lib/fontmake/font_project.py

```diff
@@ -413,12 +417,14 @@ def run_from_designspace(
 
         ufos = []
         if not interpolate or masters_as_instances:
-            reader = DesignSpaceDocumentReader(designspace_path, ufoVersion=3)
+            reader = DesignSpaceDocumentReader(designspace_path, ufoVersion=3,
+                                               round_geometry=round_geometry)
```

Comment:
I believe here passing `round_geometry` argument doesn't make any difference, as no interpolation is happening in this branch.

Reply:
Good point.